src/util.ts

```diff
@@ -88,3 +88,26 @@ export function createOutputRetainingCompilerHost(
     outputFiles.set(fileName, content);
   }
 }
+
+/**
+ * Returns the declarations for the given decorator.
```

Comment:
This function is now used by both decorator-annotator.ts and tsickle.ts.
Should tsickle import it from decorator-annotator? It's hard for me to judge which file uses it "most closely", tbh.